File: src/main/java/com/example/jobs/dto/Tag.java
package com.example.jobs.dto;

import lombok.Builder;
import lombok.Getter;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

@Getter
@Builder
@Table(name = "tags")
public class Tag {
  @Id
  @Column("tag_id")
  Integer tagId;

  String name;

  @Column("job_count")
  Integer jobCount;
}

File: src/main/kotlin/com/example/jobs/entity/Tag.kt
package com.example.jobs.entity

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column
import org.springframework.data.relational.core.mapping.Table

@Table("tags")
class Tag {
    @Id
    @Column("tag_id")
    var tagId: Int? = null

    var name: String = ""

    @Column("job_count")
    var jobCount: Int = 0
}